[CS2113T-W12-3] SLAM

alwinangys opened the pull request:
SLAM (Student Life Affairs Manager) is a desktop app for student group leaders to manage their groups events, tasks and various activities,  optimized for use via a Command Line Interface (CLI).

saravanamani1999 commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,64 @@
+
+#### Select Functionality
+
+![](images/SelectDiagram.png)
```
<img width="437" alt="Screenshot 2021-10-27 at 11 32 52 AM" src="https://user-images.githubusercontent.com/66838856/138997174-a35d89fd-b0fe-4a5c-9304-c489b262e280.png">
In the method return ,CommandResult, for execute(), should the arrow be dashed for CommandResult?

kelvneo replied:
![image](https://user-images.githubusercontent.com/2332196/139173800-b7c421dd-2f0d-4a83-9ead-9512bbb1efb7.png)

Consider putting numbering for your steps here? Since you used numberings for others as well.

leecheokfeng replied:
![image](https://user-images.githubusercontent.com/70262560/139187819-4ab2201c-b7f6-4e83-add9-75c986700cba.png)

Should the CommandResult arrow be dashed since it is returning an object?

saravanamani1999 commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,64 @@
+#### Select Functionality
+
+![](images/SelectDiagram.png)
+
```
<img width="437" alt="Screenshot 2021-10-27 at 11 32 52 AM" src="https://user-images.githubusercontent.com/66838856/138997174-a35d89fd-b0fe-4a5c-9304-c489b262e280.png">
Should the activation bar for SelectCommand end after returning CommandResult?

saravanamani1999 commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,64 @@
+3. `DeleteCommand` has 3 uses: deleting an `Event`, a `Task`, or to delete all `Event`s through the command `delete all`.
+4. The constructor processes the usage for `DeleteCommand` and executes the actual deletion through `execute` which returns a `CommandResult` object with the associated deletion message from the `Ui` class.
+5. `delete all` will not immediately invoke the `clear()` method on the global `eventCatalog` ArrayList and will instead prompt a confirmation from the user before deleting all `Event`s.
+
```
Overall tip: You could try hiding the footboxes at the bottom of each class if you want to, with `hide footbox`

saravanamani1999 commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,64 @@
+#### Save Functionality
+
+![](images/SaveDiagram.png)
+
```
<img width="903" alt="Screenshot 2021-10-27 at 11 43 20 AM" src="https://user-images.githubusercontent.com/66838856/138997308-9dab123e-1b77-4db8-94c9-60c3e22b1667.png">
Should the parameter types for all the methods be included? Because it is present in the method `save()` but not for the rest.

lixiyuan416 replied:
I think you diagram will be clearer if you don't include the method parameters!
Are they really needed to explain the flow of method calls of the storage functionality?

saravanamani1999 commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,64 @@
+#### Load Functionality 
+
+![](images/LoadDiagram.png)
+
```
<img width="704" alt="Screenshot 2021-10-27 at 11 48 33 AM" src="https://user-images.githubusercontent.com/66838856/138997504-23db88a5-d6d5-4284-8d8e-7db1fdf36d5d.png">

Should the class entities `File` and `eventCatalog` include `:`? Also should `eventCatalog` start with a capital E since its a class name?

saravanamani1999 commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,64 @@
+#### Save Functionality
+
+![](images/SaveDiagram.png)
+
```
<img width="647" alt="Screenshot 2021-10-27 at 11 48 27 AM" src="https://user-images.githubusercontent.com/66838856/138997433-21864740-ec6b-4f92-86e8-520ef5c0efd6.png">

Should the class name start with a `:`?

kelvneo commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,97 @@
+4. `task` will display next upcoming task and `event` will display next upcoming event
+
+#### Update Functionality
+![](images/UpdateDiagram.png)
```
![image](https://user-images.githubusercontent.com/2332196/139173995-efb95953-2493-41aa-ac91-35eba5f30ff8.png)

You may want to add a return arrow from `UpdateCommand` to `Parser`

leecheokfeng replied:
![image](https://user-images.githubusercontent.com/70262560/139186436-c6300cf8-7885-439b-ab0f-cd8406c3e943.png)


Should the activation bar for UpdateCommand be attached to the UpdateCommand box? Since it is a constructor.

leecheokfeng replied:
![image](https://user-images.githubusercontent.com/70262560/139187218-b6c1a5f3-0864-4800-bd82-102cc636f7e9.png)

Are 14 and 15 method calls or method returns? The description says they are returned, if so should they be dashed lines? Or does the description need rephrasing?

kelvneo commented on docs/diagrams/UpdateDiagram.puml:
```diff
@@ -0,0 +1,44 @@
+        end
+
+    UpdateCommand -> Duke: postUpdateMessage()
+    UpdateCommand -> Duke: CommandResult(executeResult);
```
![image](https://user-images.githubusercontent.com/2332196/139174197-d29aaee0-3483-4464-b77e-ecef78cf1f6f.png)

You may want to change the arrow type to a dotted arrow using `.>` instead to indicate returning values

kelvneo commented on docs/diagrams/LoadDiagram.puml:
```diff
@@ -0,0 +1,39 @@
+        return task:Task
+        |||
+        ":StorageFile" -> ":TaskSetter": setXYZAttributes()
+        note over ":TaskSetter": Sets the currEvent for the Task,\n adds the task to the currEvent\n and assigns task to respective members
```
Nice use of notes!

kelvneo commented on docs/diagrams/LoadDiagram.puml:
```diff
@@ -0,0 +1,39 @@
+|||
+loop lines
+    alt classType != 't'
+        ":StorageFile" -> ":StorageFile": add
```
I believe you are missing a `()` for your `add` method here.

kelvneo commented on docs/diagrams/LoadDiagram.puml:
```diff
@@ -0,0 +1,39 @@
+        note over ":TaskSetter": Sets the currEvent for the Task,\n adds the task to the currEvent\n and assigns task to respective members
+        activate ":TaskSetter" #lightblue
+        return
+    end
```
You may want to standardize adding types to your arguments and return objects with your teammates.

lixiyuan416 commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,97 @@
 
 ## Design & implementation
```
Perhaps it's better to separate Design and Implementations into 2 sections?
Alternatively, perhaps you can add in subsections to make it clear that which part is documenting Design, which part is documenting Implementation in each component.

lixiyuan416 commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,97 @@
+7. `list` : to list the Overall Schedule 
+8. `list -m` : to list all the members in the Overall Members List
+9. `list {Event_Num} -t` : to display all the tasks in a unique Event
+10. `list {Event_Num} t/{Task_Num}` : to display all the members involved in a specific task
```
Maybe this shouldn't be included here as you probably have covered command syntax in the UG

DmitriYam replied:
![image](https://user-images.githubusercontent.com/69571609/139373033-4d19b1a6-2203-4b77-9ff7-c0a4ac3a02c3.png)
The list suffixes could be put into a more reader friendly forma such as a table.

lixiyuan416 commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,97 @@
+4. `task` will display next upcoming task and `event` will display next upcoming event
+
+#### Update Functionality
+![](images/UpdateDiagram.png)
```
![image](https://user-images.githubusercontent.com/69444693/139186752-f8454e41-b909-4565-aafb-f0c2f11356f8.png)
You may want to make it clearer what happens in the sequence diagram if `!isError`.
Right now it seems that the alt loop happens regardless of the value of `isError`

yapjoonsiong replied:
activation bar should connect to UpdateCommand box since it is being constructed. Might wanna add a return arrow from UpdateCommand to Parser as well
![Capture](https://user-images.githubusercontent.com/70307456/139370357-cece3277-a3eb-48ec-b45d-602cdbf807da.PNG)

yapjoonsiong replied:
![Capture](https://user-images.githubusercontent.com/70307456/139370503-072dca0d-6cb6-4dd2-ac3a-294edef8dccb.PNG)
return arrows should be dashed?

DmitriYam replied:
![image](https://user-images.githubusercontent.com/69571609/139372490-6956b665-c29c-44a4-a518-fb181d102615.png)
You may want to make new activation bars to represent points 6 and 8 in the diagram to make the clearer.

lixiyuan416 reviewed (commented):
You might want to not go in depth into the nitty gritty of how every single functionality works, what method they call etc.

Consider using a higher level overview for some functionalities (eg architecture diagrams)

leecheokfeng commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,97 @@
+
+#### Load Functionality 
+
+![](images/LoadDiagram.png)
```
![image](https://user-images.githubusercontent.com/70262560/139188210-cfdc40f2-259d-424b-9cfc-c5a89713cdc3.png)

Would it be clearer to use [else] as the condition instead of [classType == 't']?

leecheokfeng reviewed (commented):
Good effort

DmitriYam commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,97 @@
+   3. The past added event will be set to the `event` attribute of the new `task`, and the `task` will be added to the list of tasks in `currEvent`. The `task` will also be assigned to `Member` objects in `MemberRoster` if the `Member` objects are assigned to the `task` (bidirectional association) (see Steps 12-13 in diagram).
+
+The `load` functionality asserts the following based on **SLAM**'s core design principles:
+- No tasks can exist without a member assigned to it, but a member can exist without an assigned task. As such, member objects will be first created and added to the roster when loading ([refer to Save functionality to understand how this is achieved](#save-functionality)).
```
![image](https://user-images.githubusercontent.com/69571609/139371306-376dbf75-66cb-430a-aa23-ea350ec03c8d.png)

The numbering of the arrows clutters up the diagram maybe you want to not include them?

yapjoonsiong commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,97 @@
+
+#### Select Functionality
+
+![](images/SelectDiagram.png)
```
![Capture](https://user-images.githubusercontent.com/70307456/139371448-d64707a7-9910-4ea6-ad59-9220e63adbd7.PNG)
CommandResult return statement, should be dashed arrow?

DmitriYam commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,97 @@
+#### Select Functionality
+
+![](images/SelectDiagram.png)
+
```
![image](https://user-images.githubusercontent.com/69571609/139371769-b735cd03-9200-4033-9665-a76f33d77713.png)
The result should be a dotted arrow and the activation bar should end as selectcommand is no longer in control of the porogram

yapjoonsiong commented on docs/DeveloperGuide.md:
```diff
@@ -6,8 +6,97 @@
+3. Then it would initialize the `listType` value depending on the `userCommand`
+4. This processed information would be passed back to `parser` then on to `Duke`.
+5. `Duke` then calls the `execute` method in `ListCommand` which will then return an object of type `CommandResult` and would print out the list corresponding to the `listType` in chronological order
+6. list can display 4 types of list depending on the `listType` 
```
might want to omit point 6 onwards as they do not explain how the code works, only syntax.

DmitriYam reviewed (commented):
W11 tut review

antrikshv committed:
List Command refactored

antrikshv committed:
NextCommand refactored

antrikshv committed:
Junit and Checkstyle fixed

antrikshv committed:
help command summary updated

antrikshv committed:
updated update task intromessage

antrikshv committed:
task intro message updated

antrikshv committed:
updated implementation to update parser

antrikshv committed:
removed exception in command class

antrikshv committed:
update commands fixed

antrikshv committed:
checkstyle fixed

antrikshv committed:
fix

antrikshv committed:
checkstyle fix

antrikshv committed:
Junit test updated for update list and next

Alvinlj00 committed:
Merge pull request #184 from antrikshv/master

Updated the parser commands for List, Next, Update and handle majority of bugs

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

Alvinlj00 committed:
Merge branch 'master' into alvinLim-SplitAddCommands

alwinangys committed:
minor improve code

alwinangys committed:
restart SelectCommand

alwinangys committed:
add ability to select event and task

crabnuggets committed:
crabnuggets.md: Add further updates to ppp

alwinangys committed:
improve code

crabnuggets committed:
Add event of index and event title to task encoding

crabnuggets committed:
Add event index and title decoding for tasks

crabnuggets committed:
Add parseDateTimeForSaving method

* Does not throw exception
* Does not care about whether date has already passed since
such invalid data cannot be saved in the first place

Alvinlj00 committed:
Modify AddParser and add specific AddCommand files

Alvinlj00 committed:
Fix checkstyle errors and bugs

Alvinlj00 committed:
Fix checkstyle errors

alwinangys committed:
update parsing and searching for member

Alvinlj00 committed:
Add comparator for items

Alvinlj00 committed:
Merge pull request #187 from Alvinlj00/alvinLim-SplitAddCommands

Update AddParser and split AddCommand for more OOP

alwinangys committed:
add execution of selected member via index

alwinangys committed:
add select member mesesage

alwinangys committed:
add parsing of selecting member

alwinangys committed:
Merge branch 'master' into v2.1-select-1

alwinangys committed:
improve print messages

alwinangys committed:
improve error message

alwinangys committed:
remove whitespace

alwinangys committed:
fix catching invalid task index bug

alwinangys committed:
fix checkstyle

alwinangys committed:
initial commit

alwinangys committed:
add ability to find tasks

alwinangys committed:
Merge pull request #185 from alwinangys/v2.1-select-1

New SelectCommand for Events, Tasks, Members

alwinangys committed:
condense find results directly into string

crabnuggets committed:
Update file saving to catch errors and print errors in loading

crabnuggets committed:
Merge branch 'master' into 2.1Update-Storage

alwinangys committed:
add JUnit tests

alwinangys committed:
Merge branch 'master' into v2.1-find-1

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

alwinangys committed:
fix checkstyle

alwinangys committed:
remove whitespace

alwinangys committed:
comment out code to postpone fix

alwinangys committed:
fix indentation

crabnuggets committed:
Update task decoder to check for validity of members

crabnuggets committed:
Update JUnit tests

crabnuggets committed:
Clean up imports and code

crabnuggets committed:
Fix member decoder JUnit test

crabnuggets committed:
Improve branch coverage of JUnit tests

crabnuggets committed:
Update EXPECTED.TXT

crabnuggets committed:
Refactor name of load method to be slightly more descriptive

alwinangys committed:
refactor filtering of events into FindCommand

alwinangys committed:
improve JUnit

alwinangys committed:
add more JUnit

alwinangys committed:
improve JUnit

alwinangys committed:
improve JUnit

alwinangys committed:
Merge pull request #190 from alwinangys/v2.1-find-1

New FindCommand with JUnit tests

Alvinlj00 committed:
Update DoneUndoParser and DoneUndoCommand

Alvinlj00 committed:
Resolve merge conflicts

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

Alvinlj00 committed:
Merge branch 'master' into alvinLim-DoneUndoParser

Alvinlj00 committed:
Fix checkstyle errors

Alvinlj00 committed:
Fix UI issues

Alvinlj00 committed:
Update UI for done and undo commands

antrikshv committed:
update to listparser

antrikshv committed:
Merge branch 'master' of https://github.com/antrikshv/tp into additional-pull

crabnuggets committed:
Merge pull request #192 from crabnuggets/2.1Update-Storage

Upgrade `storage` package defensiveness

Alvinlj00 committed:
Merge branch 'master' into alvinLim-DoneUndoParser

Alvinlj00 committed:
Merge pull request #193 from Alvinlj00/alvinLim-DoneUndoParser

Update DoneUndoParser and DoneUndoCommand

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

alwinangys committed:
Merge branch 'master' into v2.1-delete-1

alwinangys committed:
restart deleteCommand

alwinangys committed:
deleteParser initial commit

alwinangys committed:
improve code

alwinangys committed:
first test for deleting event and task

alwinangys committed:
fix IndexOutOfBounds exception

alwinangys committed:
fix IndexOutOfBounds exception

alwinangys committed:
improve code

alwinangys committed:
remove commented out code

alwinangys committed:
improve error messages

crabnuggets committed:
Update .puml and image files for load and save diagrams

crabnuggets committed:
Update DG for load and save functionalities

alwinangys committed:
improve code

alwinangys committed:
add initial deletemember functionality

crabnuggets committed:
Merge branch 'master' into PPP-crabnuggets

alwinangys committed:
improve code

antrikshv committed:
added the ability to display calendar with next -e

antrikshv committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	src/main/java/seedu/duke/Ui.java

crabnuggets committed:
Update PPP

crabnuggets committed:
Fix bug whereby task decoder was creating new duplicate member objects

antrikshv committed:
gradlew checkstyle fixed

antrikshv committed:
Merge pull request #199 from crabnuggets/v2.1-UpdateDG

Update DG - `save` and `load` functionaltiy

antrikshv committed:
Merge pull request #198 from crabnuggets/PPP-crabnuggets

PPP - crabnuggets

antrikshv committed:
Merge pull request #196 from antrikshv/master

Added the ability to display calendar with the date of the event highlighted

antrikshv committed:
Merge pull request #195 from alwinangys/v2.1-delete-1

Implemented new DeleteCommand

antrikshv committed:
Merge pull request #201 from crabnuggets/v2.1-StorageFileFix

Fix bug whereby task decoder was creating new duplicate member objects

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

alwinangys committed:
fix bugs

alwinangys committed:
first PPP commit

alwinangys committed:
Merge pull request #202 from alwinangys/v2.1-select-bugs

Fixed DeleteCommand bugs

alwinangys committed:
update PPP

alwinangys committed:
Merge branch 'master' into alwinAng-PPP-1

alwinangys committed:
Merge pull request #204 from alwinangys/alwinAng-PPP-1

PPP - alwinangys

Alvinlj00 committed:
Add DoneUndoCommand JUnit tests

Alvinlj00 committed:
Add AddCommand JUnit tests

Alvinlj00 committed:
Fix bug where adding an incorrect date format does not throw exception

Alvinlj00 committed:
Fix checkstyle errors

Alvinlj00 committed:
Merge pull request #205 from Alvinlj00/alvinLim-JUnitTests

Add JUnit Tests for AddCommand and DoneUndoCommand

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

antrikshv committed:
dg updates

crabnuggets committed:
Update NextParser and MemberParser to abstract classes

antrikshv committed:
delete bugs fixed

antrikshv committed:
merge conflict resolved

antrikshv committed:
Merge branch 'master' into master

antrikshv committed:
Merge pull request #1 from AY2122S1-CS2113T-W12-3/master

added style to dg

crabnuggets committed:
Parser.java: Add additional flag check for retrieveItemAttribute

crabnuggets committed:
Fix exception message typo

crabnuggets committed:
Fix formatting of exception messages

alwinangys committed:
fix bug

alwinangys committed:
Merge pull request #207 from alwinangys/v2.1-delete-bug

FIx DeleteCommand bug

crabnuggets committed:
Merge branch 'master' into v2.1-BugFixes

crabnuggets committed:
Fix parsing bug

Fix bug where inputting garbage string that ends with the
appropriate flag is considered as a valid flag.

E.g. asdfn/ is considered as a valid flag for title

Users are now required to separate flags by at least a whitespace

crabnuggets committed:
Parser.java: Optimize imports

crabnuggets committed:
Abstract previous bug fix to only within AddParser

The previous bug fix causes issue with the Update command as
item attributes in that command will contain no whitespace prior
to the flag. The fix has been abstracted to a new method to be used
only with AddParser.

crabnuggets committed:
Update convertDateTime method

Add new ability to handle additional whitespaces between the given
date and time.

E.g. "19-02-2022    2359" can now be parsed properly

crabnuggets committed:
Parser.java: Add additional trim to retrieveItemAttribute method

antrikshv committed:
Merge branch 'AY2122S1-CS2113T-W12-3:master' into master

antrikshv committed:
DG Design added

antrikshv committed:
Merge branch 'master' of https://github.com/antrikshv/tp into PR1

crabnuggets committed:
Parser.java: Update extractInt method and rename into extractIndexes

Rename extractInt to extractIndexes for better descriptiveness.
Previously extractIndexes would extract duplicate indexes as well.
Chang return type of method to ArrayList<Integer> such that
duplicates can be caught and will not be returned.

crabnuggets committed:
Update date formatter to catch invalid Feb dates

crabnuggets committed:
Fix UI message formatting.

alwinangys committed:
update UG

crabnuggets committed:
Fix checkstyle

alwinangys committed:
Merge pull request #208 from antrikshv/master

Added introduction, design and implementation into DG

alwinangys committed:
Merge branch 'master' into v2.1-ug-1

alwinangys committed:
Merge pull request #209 from alwinangys/v2.1-ug-1

Update UG

antrikshv committed:
DG updates

antrikshv committed:
Merge branch 'PR1'

alwinangys committed:
update aboutUs.md

crabnuggets committed:
Update extractIndexes to only split via whitespace

Additionally, fixed some exception message format errors as a result
of this change.

crabnuggets committed:
Update JUnit tests

crabnuggets committed:
Remove additional whitespace in Ui.updateIntroMessage

crabnuggets committed:
Fix some minor Ui bugs with update command

crabnuggets committed:
Add acknowledgements

crabnuggets committed:
Update command type to be case insensitive

antrikshv committed:
Added hyperlinks, product scope, glossary, non-functional requirements

antrikshv committed:
Bug fixed for next command

Alvinlj00 committed:
Update DG Add and Done Undo Commands

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

Alvinlj00 committed:
Merge branch 'master' into alvinLim-Docs

# Conflicts:
#	docs/DeveloperGuide.md

alwinangys committed:
Merge pull request #212 from antrikshv/master

DG Additions and next command bug fixed

alwinangys committed:
Merge branch 'master' into alwinAng-PPP-2

antrikshv committed:
UserGuide updated

antrikshv committed:
Merge pull request #211 from crabnuggets/v2.1-BugFixes

V2.1 Bug fixes

antrikshv committed:
Merge branch 'PR1'

antrikshv committed:
small typo fixed

crabnuggets committed:
Add user stories to DG

crabnuggets committed:
Merge pull request #213 from antrikshv/master

Userguide updated

alwinangys committed:
Merge pull request #214 from crabnuggets/v2.1-UpdateDGpt2

Update DG - Acknowledgements and User Stories

Alvinlj00 committed:
Update UG for add task, done and undo task, add member functionalities

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

Alvinlj00 committed:
Merge branch 'master' into alvinLim-Docs

alwinangys committed:
update DG

alwinangys committed:
Merge branch 'master' into alwinAng-PPP-2

Alvinlj00 committed:
Merge pull request #215 from Alvinlj00/alvinLim-Docs

Update UG and DG for AddCommand and DoneUndoCommand

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

alwinangys committed:
add UML code

alwinangys committed:
Merge branch 'master' into alwinAng-PPP-2

antrikshv committed:
PPP added (antriksh)

antrikshv committed:
Merge branch 'AY2122S1-CS2113T-W12-3:master' into master

crabnuggets committed:
Update instructions for manual testing

alwinangys committed:
update DG for select

alwinangys committed:
Merge pull request #216 from crabnuggets/v2.1-UpdateDGpt3

Add instructions for manual testing

alwinangys committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp into alwinAng-PPP-2

Alvinlj00 committed:
Add PPP

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

antrikshv committed:
Merge branch 'AY2122S1-CS2113T-W12-3:master' into master

alwinangys committed:
Merge pull request #217 from alwinangys/alwinAng-PPP-2

Update Select DG

antrikshv committed:
PPP added, and diagrams for DG updated

antrikshv committed:
Merge branch 'PR1'

antrikshv committed:
Merge branch 'master' into master

antrikshv committed:
Merge pull request #2 from AY2122S1-CS2113T-W12-3/master

updated dg diagrams and PPP

Alvinlj00 committed:
Change profile picture

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

Alvinlj00 committed:
Merge branch 'master' into alvinLim-PPP

Alvinlj00 committed:
Remove dummy file

Alvinlj00 committed:
Merge pull request #218 from Alvinlj00/alvinLim-PPP

Add PPP documentation

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

antrikshv committed:
Add/DoneUndo Diagrams updated

Alvinlj00 committed:
Merge pull request #219 from antrikshv/master

Updated Dg diagrams and PPP

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp into alvinLim-DGChanges

Alvinlj00 committed:
Change done and undo implementation explanation

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

Alvinlj00 committed:
Merge branch 'master' into alvinLim-DGChanges

antrikshv committed:
Dashboard calendar added

antrikshv committed:
Merge branch 'AY2122S1-CS2113T-W12-3:master' into master

crabnuggets committed:
Update DG for manual testing - tasks

crabnuggets committed:
Merge branch 'master' into v2.1-UpdateDGpt3

crabnuggets committed:
Merge pull request #220 from crabnuggets/v2.1-UpdateDGpt3

Update DG Manual Testing - Tasks

alwinangys committed:
Update DG

alwinangys committed:
Merge branch 'master' into alwinAng-DG-1

Alvinlj00 committed:
Update Load and Save diagrams

Alvinlj00 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-3/tp

Alvinlj00 committed:
Merge branch 'master' into alvinLim-DGChanges

Alvinlj00 committed:
Merge pull request #221 from Alvinlj00/alvinLim-DGChanges

Modify Load and Save diagrams

alwinangys committed:
update DG

alwinangys committed:
Merge branch 'master' into alwinAng-DG-1

alwinangys committed:
Merge pull request #222 from alwinangys/alwinAng-DG-1

Update DG

Alvinlj00 committed:
Fix AboutUs page

antrikshv committed:
dashboard calendar reverted

Alvinlj00 committed:
Merge pull request #223 from Alvinlj00/alvinLim-AboutUsFix

Fix AboutUs Page

antrikshv committed:
Merge branch 'AY2122S1-CS2113T-W12-3:master' into master

crabnuggets committed:
Update UG

crabnuggets committed:
Update README.md

crabnuggets committed:
Update storage to save after every command

crabnuggets committed:
Merge branch 'master' into v2.1-UpdateUG

crabnuggets committed:
Merge pull request #224 from crabnuggets/v2.1-UpdateUG

Update UG Command Summary and Implement File Saving After Every Command

antrikshv committed:
Merge branch 'AY2122S1-CS2113T-W12-3:master' into master

crabnuggets committed:
Update internal links for DG

alwinangys committed:
Merge pull request #225 from antrikshv/master

Dg diagram and hyperlinks fixed

crabnuggets committed:
Merge branch 'master' into v2.1-UpdateUG

crabnuggets committed:
Merge pull request #226 from crabnuggets/v2.1-UpdateUG

V2.1 update DG internal links

antrikshv committed:
add and doneundo hyperlinks added

antrikshv committed:
Merge branch 'master' of https://github.com/antrikshv/tp

# Conflicts:
#	docs/DeveloperGuide.md

antrikshv committed:
typo fixed

crabnuggets committed:
no message

crabnuggets committed:
Merge pull request #228 from crabnuggets/v2.1-UpdateUG

Update DG

antrikshv committed:
Merge branch 'master' into master

crabnuggets committed:
Update UG and DG

crabnuggets committed:
Merge branch 'master' into v2.1-UpdateUG

crabnuggets committed:
Merge pull request #229 from crabnuggets/v2.1-UpdateUG

V2.1 update ug

crabnuggets committed:
Update UG

crabnuggets committed:
Merge branch 'master' into v2.1-UpdateUG

crabnuggets committed:
Merge pull request #230 from crabnuggets/v2.1-UpdateUG

V2.1 update ug

crabnuggets committed:
Update UG

crabnuggets committed:
Merge branch 'master' into v2.1-UpdateUG

crabnuggets committed:
Merge pull request #231 from crabnuggets/v2.1-UpdateUG

V2.1 update ug

crabnuggets committed:
update ug

crabnuggets committed:
Merge pull request #232 from crabnuggets/v2.1-UpdateUG

update ug

crabnuggets committed:
UPDATE UG

crabnuggets committed:
Merge pull request #233 from crabnuggets/v2.1-UpdateUG

UPDATE UG
